src/notification/__tests__/notification-test.js

```diff
@@ -11,6 +11,8 @@ import * as eg from '../../__tests__/lib/exampleData';
 import { fromAPNsImpl as extractIosNotificationData } from '../extract';
 import objectEntries from '../../utils/objectEntries';
 
+const realm_uri = eg.realm.href;
```

Comment:
Tiny nit: I've slightly preferred `.toString()` over `.href`. What do you think?

We're asking for a different representation of the whole URL, and I think `.toString()` is a bit clearer about that. `.href` looks a little bit like `.origin`, `.hostname`, and friends: the things you'd use to pull out just a component of the URL.

Reply:
Sure, that's probably a bit nicer. Changing that and merging.